integrations/analytics.js-integration-tiktok-pixel/lib/index.ts

```diff
@@ -96,7 +114,31 @@ TikTokPixel.prototype.loaded = function (): boolean {
   return !!(ttq() && ttq()._i[this.options.pixelId]._load);
 };
 
-// TODO: Add tracking implementations
+/**
+ * Trigger a page view.
+ */
+TikTokPixel.prototype.page = function (_page: Page): void {
+  ttq().page();
```

Comment:
Does the `page` event need to us the `trackContext` in any way to get the `event_id` or pass in any properties from the Page object?

Reply:
I don't see any documentation around their `page` API and what properties we can pass, but it's just a generic page tracking function and it automatically passes along the url from the browser. It gets fired on every page.

But marketing cares about the event `ViewContent`, which will be fired via the `track` function, when they view a PDP. That will be tracked separately from the generic page view tracking, and that `ViewContent` event will have the properties about the product and also include the `event_id` for the backend Events API deduping